solutions/swb-reference/integration-tests/tests/isolated/environmentTypeConfigs/create.test.ts

```diff
@@ -158,6 +158,7 @@ describe('create environment type configs', () => {
       checkHttpError(
         e,
         new HttpError(403, {
+          statusCode: 403,
```

Comment:
Why is there a status code here?

Reply:
This shouldn't be a 403, it should be a 400 right? Its a missing value that is required, not an unauthorized access

Reply:
Apparently this is an existing issue. I'm not going to block the PR on it.